lib/core/src/display/shape/text/glyph/system.rs

```diff
@@ -130,12 +125,23 @@ impl Line {
 #[allow(missing_docs)]
 impl Line {
     /// The starting point of this line's baseline.
-    pub fn baseline_start(&self) -> &Vector2<f32> { &self.baseline_start }
+    pub fn baseline_start(&self) -> &Vector2<f32> {
+        &self.baseline_start
+    }
+
     /// Line's height in pixels.
-    pub fn height        (&self) -> f32           { self.height          }
+    pub fn height(&self) -> f32 {
+        self.height
+    }
+
     /// Number of glyphs, giving the maximum length of displayed line.
-    pub fn length        (&self) -> usize         { self.glyphs.len()    }
-    pub fn font_id       (&self) -> FontId        { self.font_id         }
+    pub fn length(&self) -> usize {
+        self.glyphs.len()
+    }
+
+    pub fn font(&self) -> FontHandle {
```

Comment:
Missing docs. + GH tells about linting issues. Was this reported by linter? If not, could you please investigate why?

Reply:
Please, in one of previous PR you said, that the docs for getters are not required and you alone proposed to add `#[allow(missing_docs)]` to such blocks.

So I added and removed the docs which did not give more information than function name.